docs: mention "london" among accepted values for "evmVersion" (#1778)
nomiclabs/hardhat: docs/guides/compile-contracts.md

- `optimizer`: an object with `enabled` and `runs` keys. Default value: `{ enabled: false, runs: 200 }`.

> - `evmVersion`: a string controlling the target evm version. One of `homestead`, `tangerineWhistle`, `spuriousDragon`, `byzantium`, `constantinople`, `petersburg`, `istanbul`, `berlin` or `london`. Default value: managed by `solc`.

Reviewer: ```suggestion
- `evmVersion`: a string controlling the target evm version. For example: `istanbul`, `berlin` or `london`. Default value: managed by `solc`.
```
